docs/content/users/extend/traefik-router.md

```diff
@@ -55,16 +55,8 @@ Project-specific configuration is automatically generated in the project’s `.d
 * The `certs` directory contains the `<projectname>.crt` and `<projectname>.key` certificate generated for the project.
 * The `config/<projectname>.yaml` file contains the configuration for the project, including information about routers, services, and certificates.
     * If you alter a `config/<projectname>.yaml` file, you need to remove the `#ddev-generated` from the top of the file, so it won't be overwritten on the next `ddev start`.
```

Comment:
Let's just not suggest this. People may be familiar with the technique and they'll know what to do, and it will work.
```suggestion
```